apps/AppWithWearable/lib/backend/database/memory.dart

```diff
@@ -40,6 +42,7 @@ class Memory {
     this.recordingFilePath,
     this.startedAt,
     this.finishedAt,
+    this.coordinates,
```

Comment:
Let's create a location object instead,
and inside let's add the coordinates list.

So that later we can have more stuff about the location.